cves/CVE-2004-0747.yml

```diff
@@ -42,17 +46,22 @@ bounty:
   announced:
   url:
 reviews: []
-bugs: []
+bugs: [11182]
 repo:
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-   - commit: e29f8e261960e178d08c4e0a2f82058c0835e512
-     note:
+   - commit: f2d3d37f2dd9e1c31efff9e08c6508f1c6198318
+     note: |
+      notroj had to rewrite the ap_resolve_env function to avoid buffer overflow. The rewritten function
+      uses a SMALL_EXPANSION (set to 5) variable which the function checks to make sure expansion does not occur 
+      beyond this to avoid obverflow. It also casts the ap_resolve_env function to (char *).
 vccs:
-  - commit: 09736d5a1a08decacae94e2260717d9a8dc03ed2
-    note:
+  - commit: 99a9d6e3e02dcd0d996097e8c4d23a39c5dbbf44
+    note: |
+      Orginial commit of the ap_resolve_env function that claims that the change for handling ${ENV} constructs
```

Comment:
Original *

Reply:
You could also just write environments instead of ${ENV}

Reply:
I'll make that change.